### DISCUSSION TITLE
Git contributions doesn't show on personal account after merging work account

### DISCUSSION THREAD

**Discussion by manjeetk** [community/community#36369] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi,
I have followed the steps as mentioned below to merge my work account to personal github account. But none of the contributions from the work account came through to the personal account and now work account repos show contributions from the ghost account.
It doesn't consider my personal account even though I have added my work email address to the personal GitHub account.

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts

Could anyone please help how to resolve this?

Thank you

**Comment by airtower-luna** [comment, marked as answer]
> But none of the contributions from the work account came through to the personal account and now work account repos show contributions from the ghost account.

What kind of contributions do you mean? "Ghost account" sounds like you might refer to things like issues, PRs, etc. Those won't be reassigned. Adding the email address to your current account should ensure that *commits* are recognized as your contributions, unfortunately that doesn't apply to other things.

**Reply by manjeetk** [reply]
Thanks for your reply @airtower-luna  
As per the github docs link,  I assumed it will transfer contributions to the personal account. But seems like you are right. It won't reassign.

**Comment by Lotusibertine** [comment]
My application for student certification has been unsuccessful, and I have tried all kinds of certification materials. The picture below is either a positioning problem or a personal profile problem, which has been modified for many times.
![image](https://user-images.githubusercontent.com/116322926/197953257-b68e34f3-3aa8-4821-97d0-f1cb2219d56f.png)

**Reply by queenofcorgis** [reply]
@Lotusibertine - you'll have a better chance of a response if you create your own post in our [Education category](https://github.com/orgs/community/discussions/categories/github-education) that way the right folks have eyes 👀 on it.
